content/en/getting_started/logs/_index.md

```diff
@@ -0,0 +1,223 @@
+
+{{< img src="getting_started/logs/plain_text_log.png" alt="Custom telnet" responsive="true">}}
+
+or a JSON object that is automatically parsed by Datadog:
+
+{{< tabs >}}
+{{% tab "US Site" %}}
+
+```
+telnet intake.logs.datadoghq.com 10514
+
+<DATADOG_API_KEY> {"message":"JSON formatted log sent through TCP", "ddtags":"env:dev", "ddsource":"terminal", "hostname":"gs-hostame", "service":"user"}
+```
+
+{{% /tab %}}
+{{% tab "EU Site" %}}
+
+```
+telnet tcp-intake.logs.datadoghq.eu 1883
+
+<DATADOG_API_KEY> {"message":"JSON formatted log sent through TCP", "ddtags":"env:dev", "ddsource":"terminal", "hostname":"gs-hostame", "service":"user"}
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+This produces the following result in your [Log Explorer Page][2]:
+
+{{< img src="getting_started/logs/json_log.png" alt="JSON logs" responsive="true">}}
+
+## Send logs from a file
+
+### Install the Agent
+
+To install the Datadog Agent within your Vagrant host, use the [one line install command][6] updated with your [Datadog API key][7]:
+
+{{< tabs >}}
+{{% tab "US Site" %}}
+
+```
+DD_API_KEY=<YOUR_DD_API_KEY>  bash -c "$(curl -L https://raw.githubusercontent.com/DataDog/datadog-agent/master/cmd/agent/install_script.sh)"
+```
+
+{{% /tab %}}
+{{% tab "EU Site" %}}
+
+```
+DD_API_KEY=<YOUR_DD_API_KEY> DD_SITE="datadoghq.eu" bash -c "$(curl -L https://raw.githubusercontent.com/DataDog/datadog-agent/master/cmd/agent/install_script.sh)"
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+#### Validation
+
+Verify the Agent is running with the [status command][8]:
+
+```
+sudo datadog-agent status
+```
+
+After a few minutes, verify the Agent is connected to your account by checking the [Infrastructure List][9] in Datadog.
+
+### Enable Log collection
+
+To enable Log collection with the Agent, edit your `datadog.yaml` [configuration file][10] located at `/etc/datadog-agent/datadog.yaml` and set `logs_enabled:true`:
+
+```yaml
+## @param logs_enabled - boolean - optional - default: false
+## Enable Datadog Agent log collection by setting logs_enabled to true.
+#
+logs_enabled: true
+```
+
+### Monitor a custom file
+#### Create the log file
+
+In order to collect logs from a custom file, first create the file and add one line of log to it:
+
+```
+$ touch log_file_to_monitor.log
+
+$ echo "First line of log" >> log_file_to_monitor.log
+```
+
+#### Configure the Agent
+
+To specify to the Agent to monitor this log file:
+
+1. Create a new configuration folder in the [configuration directory of the Agent][11]:
+
+    ```
+    sudo mkdir /etc/datadog-agent/conf.d/custom_log_collection.d/
+    ```
+
+2. Create your configuration file within this new configuration folder:
+
+    ```
+    sudo touch /etc/datadog-agent/conf.d/custom_log_collection.d/conf.yaml
+    ```
+
+3. Copy and Past the following content within this `conf.yaml` file:
+
+      ```
+      logs:
+        - type: file
+          path: /home/ubuntu/log_file_to_monitor.log
+          source: custom
```

Comment:
So here the issue is that we do not really explain what `source`, `service` and `sourcecategory` are for.

1. I would avoid using sourcecategory as this is totally optional and its behaviour might change
2. source I would suggest something like `custom_integration`
3. service something like `application1`

WDYT?

Reply:
i like it, i'm also going to link to log collection page for further reading about those parameters.

Reply:
well, now that i'm reading it, we might want to just remove the sourcecategory since in the context of this GS, the source is custom and the service is the user 😸

Reply:
yes, let's remove sourcecategory